collect_app/src/main/res/layout/first_launch_layout.xml

```diff
@@ -48,7 +49,7 @@
 
             <androidx.constraintlayout.widget.ConstraintLayout
```

Comment:
What do you think about flattening this out so we don't nested constraint layouts? I'm not seeing a reason to nest them here and doing so has a (probably very small) performance cost and, for me, makes the layout preview harder to understand.